samples/bluetooth/peripheral_with_multiple_identities/README.rst

```diff
@@ -0,0 +1,190 @@
+.. _bt_peripheral_with_multiple_identities:
+
+Bluetooth: Peripheral with multiple identities
+##############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The sample demonstrates how to use a single physical device to create and manage multiple advertisers, making it appear as multiple distinct devices.
+This can be used to test a central device that requires connections to multiple peripheral devices when you don't have many development kits available.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+Each Bluetooth device is identified by its identity address.
+When a peripheral is advertising, it uses this identity address or generates an address from this address using an identity resolving key (IRK).
+The sample application starts multiple connectable advertisers, each with its own identity, allowing a single device to advertise as multiple devices.
+
+The sample is designed to:
+#. Utilise a single physical device to emulate multiple Bluetooth advertisers.
+#. Each advertiser is assigned a unique identity and a corresponding random static address, ensuring they can be distinctly identified.
+#. Advertisers are restarted upon disconnection.
+
+Configuration
+*************
+
+|config|
+
+Configuration options
+=====================
+
+Check and configure the following Kconfig options:
+
+.. _CONFIG_BT_ID_MAX:
+
+CONFIG_BT_ID_MAX
+   This configuration defines how many identities will be used.
+
+.. _CONFIG_BT_MAX_CONN:
+
+CONFIG_BT_MAX_CONN
+   This configuration defines how many connections which can be established.
+   The sample expects this configuration to be set to the same value as **CONFIG_BT_ID_MAX**.
+
+.. _CONFIG_BT_EXT_ADV_MAX_ADV_SET:
+
+CONFIG_BT_EXT_ADV_MAX_ADV_SET
+   This configuration defines how the number of advertising sets that are available.
+   The sample expects this configuration to be set to the same value as **CONFIG_BT_ID_MAX**.
+
+.. _CONFIG_BT_BUF_ACL_RX_COUNT:
+
+CONFIG_BT_BUF_ACL_RX_COUNT
+   This configuration defines the number of incomming ACL data buffers sent from controller to the host.
+   This value needs to be greater than **CONFIG_BT_MAX_CONN**.
+
+.. _CONFIG_BT_DEVICE_NAME:
```

Comment:
Can you change the Kconfig option name? Similar name has been used as a link target in another NCS doc, which breaks the doc build.

Reply:
Hi, thank you for all the feedback. I am unsure how to do this, I will ask around :)

Reply:
As discussed offline, you can get rid of the whole Configuration options section. All options are available in the Kconfig reference docs.